tests/loom.rs

```diff
@@ -0,0 +1,31 @@
+/// Run loom tests with something like
+/// ```
+/// $ RUSTFLAGS="--cfg loom" cargo test --release loom
+/// ```
+#[cfg(loom)]
```

Comment:
seems to sadly not work without the extra module and `#![cfg(loom)]`. alright, then.

Reply:
Yeah this is the "normal" way to handle running loom tests

https://docs.rs/loom/latest/loom/#running-loom-tests